rhtap-cli/rhtap-full-install.sh

```diff
@@ -81,7 +84,7 @@ install_rhtap() {
   githubapp_integration
   gitlab_integration  # you can comment it if you don't need it
   jenkins_integration # you can comment it if you don't need it
-  ./bin/rhtap-cli deploy --timeout 35m --config ./config.yaml --kube-config "$KUBECONFIG" --debug --log-level=debug
+  ./bin/rhtap-cli deploy --timeout 35m --config ./installer/config.yaml --kube-config "$KUBECONFIG" --debug --log-level=debug
```

Comment:
I prefer defining a variable `readonly config_file="installer/config.yaml"`. If there is any change in future, we just need to change it in one place